Gemfile.lock

```diff
@@ -231,5 +231,8 @@ DEPENDENCIES
   web-console (>= 3.3.0)
   webpacker
 
+RUBY VERSION
+   ruby 2.4.0p0
```

Comment:
The question isn't really about the `gemfile.lock` its more about the fact that we don't wanna lock in patch versions, it is just highlighted by the fact that we spotted this in the `gemfile.lock`.

The real issue is that the we reference `2.4.0` directly in the gemfile rather than `~> 2.4.0` or equivalent which we should be referencing.